src/_cffi_src/openssl/x509.py

```diff
@@ -369,7 +369,7 @@
 #endif
 /* Added in 1.0.2 but we need it in all versions now due to the great
    opaquing. */
-#if OPENSSL_VERSION_NUMBER < 0x10002003L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
We definitely want to keep this version (https://github.com/pyca/cryptography/commit/5d3965bbfad2364c45e3a5e4cb9cd11b87fd2d83)